.github/workflows/build-docker-images-release.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
   version-cuda:
     name: "Latest Accelerate GPU [version]"
-    runs-on: [self-hosted, docker-gpu, multi-gpu, gcp]
+    runs-on: [self-hosted, docker-gpu]
```

Comment:
do you need GPU here ? if yes also need to use [self-hosted, single-gpu, nvidia-gpu, t4, daily-ci]